Cargo.lock

```diff
@@ -6330,9 +6320,9 @@ dependencies = [
 
 [[package]]
 name = "x86"
-version = "0.37.0"
+version = "0.45.0"
```

Comment:
Why change this?

Reply:
`llvm_asm!` is replaced with `asm!` in recent nightly, so update the dependency to fix compile issue.